modules/oxide/src/less/theme/components/menubar/promotion.less

```diff
@@ -20,11 +20,14 @@
   }
 
   .tox-promotion-link {
+    align-items: unsafe center;
     background-color: @promotion-background-color;
     border-radius: 5px;
     color: @promotion-text-color;
-    display: block;
+    cursor: pointer;
```

Comment:
Is a pointer what we really want here the Tiny logo in the statusbar has a default link icon. And this is also kind of a link even if it looks like a button.

Reply:
`pointer` is a hand-with-extended-finger one, the one that’s used on links, so I see no issue here. You might be confusing it with `default` which is usually an arrow, the one that used pretty much on every screen surface that is not interactive. Or am I reading your comment wrong?

Reply:
Ahh good got confused there about the icons. See now that it has the right one. We use the default arrow for all other UI elements the finger for this one and the other promotional link so all is good.